pkg/skaffold/build/cluster/pod.go

```diff
@@ -117,10 +118,14 @@ func (b *Builder) kanikoPodSpec(artifact *latest.KanikoArtifact, tag string) (*v
 	return pod, nil
 }
 
-func env(artifact *latest.KanikoArtifact, httpProxy, httpsProxy string) []v1.EnvVar {
+func (b *Builder) env(artifact *latest.KanikoArtifact, httpProxy, httpsProxy string) []v1.EnvVar {
+	pullSecretPath := strings.Join(
```

Comment:
nit: This should probably use `path.Join()` (but not filepath.Join). I'm OK with leaving this if you want to be super explicit though.